.github/workflows/build.yml

```diff
@@ -79,10 +81,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Create preview environment infrastructure
-        env:
-          TF_VAR_infra_provider: ${{ needs.configuration.outputs.preview_infra_provider }}
         id: create
         uses: ./.github/actions/preview-create
+        env:
+          TF_VAR_infra_provider: ${{ needs.configuration.outputs.preview_infra_provider }}
```

Comment:
I'd like to contain the "configuration surface" of these actions to be input arguments so that the environment variable names and their expected values become an implementation detail. I got rid of `TF_VAR_infra_provider` in a PR today - can you update this so that `TF_VAR_with_large_vm` also just becomes an input argument that you use to set `TF_VAR_with_large_vm` in the entrypoint script.